### ISSUE TITLE
ValueError: No weights estimated; something is wrong with input data.

### ISSUE THREAD

**Issue by lyrasheu21** [douglasyao/mesc#13]
Hello, I am trying to run expscore and encountered the problem below 
...
Estimating eQTL effect sizes for gene 44 of 44: MG_PVM_MG_PVM_PDGFB_LRP1
Skipping; no SNPS around gene
Traceback (most recent call last):
  File "mesc/run_mesc.py", line 237, in <module>
    ind.get_expression_scores(args)
  File "**lyra/mesc/mesc/expscore_indiv.py", line 269, in get_expression_scores
    raise ValueError('No weights estimated; something is wrong with input data.')
ValueError: No weights estimated; something is wrong with input data.

basically all of the "Estimating eQTL effect sizes" returns "Skipping; no SNPS around gene"

I am trying to debug but nothing seems to be wrong with my input data.
Wondering if it could be due to 
1) sample names are starting with numbers (e.g 648_R01C02)  and I know in read_csv() it'll sometimes append X infront of column names. 
2) My exp values are not gene expression and are extremely small (ranging from 0.003 to -0.003), will this affect LASSO?

**Comment by lyrasheu21** [comment]
I found my error! Thank you.

**Comment by mengying0714** [comment]
could you share what is the error? I also have the same question. Thank you .
